### PR TITLE
add-blog-head-of-science

### PR DESCRIPTION
**Pull request by emilytraff**
New blog post to promote the International Day of Women in Girls in Science on 11th Feb.

### CODE REVIEW

**Comment by gemmadallmandfe** [comment]
@emilytraff Just to flag that the build has failed on this PR - we can't review until it builds successfully!

@peteryates can support you to work out what needs changing :-)

File: app/views/content/blog/the-head-of-science-inspiring-students-in-blackpool.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: The Head of Science inspiring students in Blackpool
+date: "2022-02-09"
+author: Jessica Walmsley
+images:
+an_image:
+  path: "media/images/content/blog/jessica-walmsley.jpg"
+  thumbnail_path: "media/images/content/blog/thumbnails/jessica.jpg"
+  alt: "Jessica Walmsley against a plain background"
```
Comment (peteryates) [inline app/views/content/blog/the-head-of-science-inspiring-students-in-blackpool.md:9]:
```suggestion
  jessica-walmsley:
    path: "media/images/content/blog/jessica-walmsley.jpg"
    thumbnail_path: "media/images/content/blog/thumbnails/jessica.jpg"
```

@emilytraff slight tweak needed here, the placeholder `$jessica-walmsley$` below needs to match the value above.

**Comment by peteryates** [comment]
Looks good overall @emilytraff. The one thing we need to do a tiny bit differently now is the `alt` text. We moved it to a single place to make it easier to manage.

If you take look at [this file](https://github.com/DFE-Digital/get-into-teaching-app/blob/add-blog-head-of-science/config/images.yml) the format should make sense. I'd recommend copying the last entry (lines 288-291) and then amending the file names and alt for Jessica.

More than happy to go through the steps on a call if you like.

**Comment by gemmadallmandfe** [comment]
@emilytraff the changes haven't quite worked - @peteryates should be able to advise!

File: app/views/content/blog/the-head-of-science-inspiring-students-in-blackpool.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+title: The Head of Science inspiring students in Blackpool
+date: "2022-02-09"
+author: Jessica Walmsley
+images:
+  jessica-walmsley:
+    path: "media/images/content/blog/jessica-walmsley.jpg"
+    thumbnail_path: "media/images/content/blog/thumbnails/jessica-walmsley.jpg"
+description: |-
+  Head of Science Jessica Walmsley shares how she brings her passion for science into her Blackpool classroom.
```
Comment (EllieNodder) [inline app/views/content/blog/the-head-of-science-inspiring-students-in-blackpool.md:10]: Could we maybe make this a little longer? Ideally, meta descriptions should be between 120 and 160 characters (gives the user and Google more info about the page)

Comment (emilytraff) [reply app/views/content/blog/the-head-of-science-inspiring-students-in-blackpool.md]: Hi Ellie, I've made it a bit longer - let me know if it needs changing!

Comment (EllieNodder) [reply app/views/content/blog/the-head-of-science-inspiring-students-in-blackpool.md]: @emilytraff hey Emily, sorry it's just a little too long now! How about 'Head of Science Jessica Walmsley shares how she brings her passion for science into her Blackpool classroom and why she decided to become a teacher.' ?

Comment (emilytraff) [reply app/views/content/blog/the-head-of-science-inspiring-students-in-blackpool.md]: @EllieNodder Sorry - I've made that change!

**Review by EllieNodder — APPROVED** [review]
looks good to me!

**Pull request merged**